commit-message.md

```diff
@@ -0,0 +1,36 @@
+# Guidelines to create great commit messages
+
+
+We use [GitCop](https://gitcop.com) to check that commit messages are
+properly written. The rules are the following:
+
+* The first line of a commit message, called the subject line should
+  not be more than 80 characters long.
+
+* The commit message should end with the following trailers:
+
+  ```
+  Licence: MIT
+  Signed-off-by: User Name <email@address>
+  ```
+
+  where "User Name" is the author's real name and email@address one of
```

Comment:
On Sun, Jun 07, 2015 at 10:59:27AM -0700, Christian Couder wrote:
> Yeah I think it would work, but you need to be able to prove that
> you used the pseudonym.

I'm not a pseudonym-proponent myself, but you could probably handle
this part of it with an OpenPGP key specific to your pseudonym that
you use to sign all your commits [1,2,3].  Then you just demonstrate
that you can sign things with that key to make a fairly strong
argument that you own that pseudonym.  Of course, anyone that steals
your key and cracks your passphrase could make the same argument, but
that's a much higher bar than “no, *I'm* foobar!”.

One drawback to the signed-commit route is that either @jbenet needs
to drop his “I'll just reroll your branch for you” workflow or he'll
have to toss your signatures.  I don't have a handle on how much of an
inconvenience the former would be, but if commits by a given pseudonym
were 90% signed by your key and 10% rerolled by @jbenet and unsigned,
I expect you make a convincing argument to owning the 10% too.

[1]: http://git.kernel.org/cgit/git/git.git/tree/Documentation/RelNotes/1.7.9.txt?id=v2.4.3#n56
[2]: http://git.kernel.org/cgit/git/git.git/tree/Documentation/RelNotes/2.0.0.txt?id=v2.4.3#n135
[3]: http://git.kernel.org/cgit/git/git.git/tree/Documentation/RelNotes/2.0.0.txt?id=v2.4.3#n138